### DISCUSSION TITLE
Issue with Proximus TV using UXG Max

### DISCUSSION THREAD

**Discussion by DiDzI** [fabianishere/udm-iptv#349] [Q&A]
Hello,

I have a Unifi Gateway Max.

I use Proximus TV with an Internet Box.

I configured my Internet box in bridge mode.

I followed the tutorial https://alibotte.medium.com/unifi-et-proximus-pickx-je-taime-moi-non-plus-5de8a433136c.

I applied the settings.

When I run a diagnosis, here is the information I get:

WAN Interface: eth4
WAN VLAN: 0 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 195.238.8.0/24 172.28.40.0/21 172.28.48.0/21
LAN Interfaces: br3
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
4: eth4: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
inet X.X.X.X/X scope global dynamic eth4
valid_lft 83244sec preferred_lft 83244sec
87.64.112.0/20 proto kernel scope link src 87.64.126.61
=== Service Logs ===
Sep 11 19:36:17 UXGMax udm-iptvd[24266]: RECV V2 member report from 172.16.10.105 to 224.0.0.251
Sep 11 19:36:17 UXGMax udm-iptvd[24266]: Membership report was received on a disabled interface. Ignoring.
Sep 11 19:36:18 UXGMax udm-iptvd[24266]: RECV V2 member report from 192.168.2.240 to 224.0.0.251
Sep 11 19:36:18 UXGMax udm-iptvd[24266]: Updated route entry for 224.0.0.251 on VIF #1
Sep 11 19:36:19 UXGMax udm-iptvd[24266]: RECV Membership query from 0.0.0.0 to 224.0.0.1
Sep 11 19:36:19 UXGMax udm-iptvd[24266]: RECV V2 member report from 192.168.2.1 to 224.0.0.106
Sep 11 19:36:19 UXGMax udm-iptvd[24266]: The IGMP message was from myself. Ignoring.
Sep 11 19:36:25 UXGMax udm-iptvd[24266]: RECV Membership query from 192.168.2.1 to 224.0.0.1
Sep 11 19:36:27 UXGMax udm-iptvd[24266]: RECV V2 member report from 192.168.2.1 to 224.0.0.251
Sep 11 19:36:27 UXGMax udm-iptvd[24266]: The IGMP message was from myself. Ignoring.

But the TV channels still lag.

Do you have any idea what the problem could be?

**Comment by BrentRobert** [comment]
@DiDzI where you able to resolve this? I'm having the same issue.
